Source/Core/Core/IOS/Network/KD/NetKDRequest.h

```diff
@@ -93,6 +93,8 @@ class NetKDRequestDevice : public EmulationDevice
                                                         0xc2, 0x61, 0x91, 0x72, 0xb5, 0xcb, 0x29,
                                                         0x8c, 0x89, 0x72, 0xd4, 0x50, 0xad};
 
+  static constexpr u32 DEFAULT_SCHEDULER_SPAN = 11;
```

Comment:
```suggestion
  static constexpr u32 DEFAULT_SCHEDULER_SPAN_MINUTES = 11;
```

So the timespan units are also self-evident